src/index.html

```diff
@@ -4,14 +4,175 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content=
+      "width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body class="body">
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldPersonalInfo">
+        <legend class="legend">Personal information</legend>
+        <div class="field">
+          <label>
+            Surname:
+            <input type="text" name="surname"
+              autocomplete="off"
+            >
```

Comment:
Check point №3 of code styling [here](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)
Apply all over the project
```suggestion
            <input 
              type="text" 
              name="surname"
              autocomplete="off"
            >
```